numpy/distutils/tests/test_fcompiler.py

```diff
@@ -45,37 +45,3 @@ def test_fcompiler_flags(monkeypatch):
         else:
             assert_(new_flags == prev_flags + [new_flag])
 
-
-def test_fcompiler_flags_append_warning(monkeypatch):
-    # Test to check that the warning for append behavior changing in future
-    # is triggered.  Need to use a real compiler instance so that we have
-    # non-empty flags to start with (otherwise the "if var and append" check
-    # will always be false).
-    try:
-        with suppress_warnings() as sup:
-            sup.record()
-            fc = numpy.distutils.fcompiler.new_fcompiler(compiler='gnu95')
-            fc.customize()
-    except numpy.distutils.fcompiler.CompilerNotFound:
-        pytest.skip("gfortran not found, so can't execute this test")
-
-    # Ensure NPY_DISTUTILS_APPEND_FLAGS not defined
-    monkeypatch.delenv('NPY_DISTUTILS_APPEND_FLAGS', raising=False)
-
-    for opt, envvar in customizable_flags:
-        new_flag = '-dummy-{}-flag'.format(opt)
-        with suppress_warnings() as sup:
-            sup.record()
-            prev_flags = getattr(fc.flag_vars, opt)
-
-        monkeypatch.setenv(envvar, new_flag)
-        with suppress_warnings() as sup:
-            sup.record()
-            new_flags = getattr(fc.flag_vars, opt)
-            if prev_flags:
-                # Check that warning was issued
-                assert len(sup.log) == 1
-
-        monkeypatch.delenv(envvar)
-        assert_(new_flags == [new_flag])
-
```

Comment:
While the test will no longer warn, I think it is useful to have a test for `NPY_DISTUTILS_APPEND_FLAGS`. Maybe rework this test to still run but without the warnings?

Reply:
there is such a test left, it's right above this one